ARrehab/ARrehab/Minigames/FaceGame/Floating Mask/VirtualFaceContent.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  VirtualFaceContent.swift
+//  ARrehab
+//
+//  Created by Alan Zhang on 2020/6/6.
+//  Copyright © 2020 Eric Wang. All rights reserved.
+//
+
+import Foundation
+import ARKit
+import SceneKit
+
+protocol VirtualFaceContent {
+    func update(withFaceAnchor: ARFaceAnchor)
+}
+
+typealias VirtualFaceNode = VirtualFaceContent & SCNNode
+
+// MARK: Loading Content
+
+func loadedContentForAsset(named resourceName: String) -> SCNNode {
+    let url = Bundle.main.url(forResource: resourceName, withExtension: "scn", subdirectory: "Models.scnassets")!
+    let node = SCNReferenceNode(url: url)!
```

Comment:
change to realityKit versions.

Reply:
Any specific advice on what to look at in order to implement this in RealityKit?

Reply:
Well understand the code first and then you can read apple's intro on RealityKit. Then you should be able to redesign this in RealityKit.

I think it might be as easy as replacing SCNReferenceNode with Entity. The `url` refers to a file that contains a scene. You are going to need to rebuild that. This might be a bit tricky depending on whats inside.

Reply:
https://developer.apple.com/documentation/scenekit/scnreferencenode/1522733-referenceurl

Reply:
Ok thanks! I’ll take a look